pocs/camera/canon_gphoto2.py

```diff
@@ -35,6 +35,8 @@ def connect(self):
 
         # Properties to be set upon init.
         prop2index = {
+            '/main/settings/datetime': 'now',             # Current datetime
```

Comment:
What do you think about sorting these lines lexically?

Reply:
Done. Not sure it makes it more clear but a good idea regardless.